### PR TITLE
DDP-6266 Fixed Release Requests dashboard displaying

### PR DESCRIPTION
**Pull request by PavelYakupov**

### CODE REVIEW

File: ddp-workspace/projects/ddp-cgc/src/app/components/pages/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Route } from '../../../constants/route';
 import { filter, first, tap } from 'rxjs/operators';
 import { Component, Inject, OnInit } from '@angular/core';
 import { ActivityCode } from '../../../constants/activity-code';
+import { ActivityStatusCode } from './../../../../../../ddp-rgp/src/app/constants/activity-status-code';
```
Comment (aleksey-kuznetsov) [inline ddp-workspace/projects/ddp-cgc/src/app/components/pages/dashboard/dashboard.component.ts]: That's an accidental import I suppose.

File: ddp-workspace/projects/ddp-cgc/src/app/constants/activity-status-code.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum ActivityStatusCode {
```
Comment (aleksey-kuznetsov) [inline ddp-workspace/projects/ddp-cgc/src/app/constants/activity-status-code.ts]: No need to create your own enum, you can just import it from sdk: `ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityStatusCodes.ts`

**Review by aleksey-kuznetsov — CHANGES_REQUESTED** [review]

File: ddp-workspace/projects/ddp-cgc/src/app/components/pages/dashboard/dashboard.component.ts (hunk 1 above)
Comment (PavelYakupov) [reply ddp-workspace/projects/ddp-cgc/src/app/components/pages/dashboard/dashboard.component.ts]: It is, fixed

File: ddp-workspace/projects/ddp-cgc/src/app/constants/activity-status-code.ts (hunk 2 above)
Comment (PavelYakupov) [reply ddp-workspace/projects/ddp-cgc/src/app/constants/activity-status-code.ts]: Fixed

**Review by aleksey-kuznetsov — APPROVED** [review]

**Pull request merged**